src/main/java/it/gov/pagopa/pu/worker/WorkerApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = "it.gov.pagopa.payhub.activities")
```

Comment:
you have to configure all packages or a common base package
```suggestion
@ComponentScan(basePackages = "it.gov.pagopa")
```